### PR TITLE
Initial structure for posting to Maven Central repository

### PR DESCRIPTION
**Pull request by cotnic**
Added the initial structure to the pom.xml file for releasing to Maven Central repository.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,24 +4,56 @@
...
     </parent>
 
+    <name>SloTex NLP Entity</name>
+    <description>A library which contains POJOs for representation of all the objects stored in the database.</description>
+    <url>http://www.slotex.si</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://www.opensource.org/licenses/mit-license.php</url>
```
Comment (usatko) [inline pom.xml]: @cotnic We should use Apache licence.

File: pom.xml [hunk 2]
```diff
@@ -32,4 +64,48 @@
             <artifactId>spring-boot-starter-data-redis</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
```
Comment (usatko) [inline pom.xml:69]: :+1:

**Review by usatko — CHANGES_REQUESTED** [review]
Well done.

File: pom.xml (hunk 1 above)
Comment (cotnic) [reply pom.xml]: Changed the license.

**Pull request merged**